packages/app-mobile/components/Dropdown.js

```diff
@@ -87,6 +86,7 @@ class Dropdown extends React.Component {
 		if (this.props.labelTransform && this.props.labelTransform === 'trim') headerLabel = headerLabel.trim();
 
 		const closeList = () => {
+			this.props.hasOwnProperty('onClose') && this.props.onClose();
```

Comment:
That would crash if onClose is set to null or undefined. `if (this.props.onClose) this.props.onClose()`